academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,84 @@
+
+<HighlightBox type="info">
+
+IBC solves a widespread problem, namely cross-chain communication. This problem exists on public blockchains when exchanges wish to perform swaps. The problem emerges earlier in the case of application-specific blockchains where every asset is likely to emerge from its own purpose-built chain. Cross-chain communication is also a challenge in the world of private blockchains in cases where communication with a public chain or other private chains is desirable. There are already [IBC implementations for private blockchains such as Hyperledger Fabric and Corda](https://www.hyperledger.org/blog/2021/06/09/meet-yui-one-the-new-hyperledger-labs-projects-taking-on-cross-chain-and-off-chain-operations).
+
+</HighlighBox>
+
+## Internet of blockchains
+
+IBC is essential for application-specific blockchains like Cosmos. It offers a standard communication channel for applications on two different chains that need to communicate with each other.
+
+More details on the specifications will follow in the next section, but notice that IBC is not limited to Cosmos blockchains. Solutions can even be found for cases where some requirements are not initially met. An example of such cases is the use of IBC for Proof-of-Work (PoW) blockchains like Ethereum, as a PoW consensus algorithm which does not ensure finality. Ethereum-compatibility is enabled by creating a peg-zone where probabilistic finality is considered deterministic (i.e. irreversible) after a given threshold of block confirmations.
+
+![Internet of blockchains](images/internetofchains.png)
+
+Although application-specific blockchains offer superior scalability compared to general-purpose blockchain platforms, smart contract development for general-purpose chains and generic virtual machines (VMs), e.g. Ethereum, offer their own benefits. IBC provides a method of incorporating the strengths of general-purpose and application-specific blockchains into unified overall designs. It allows, for example, a Cosmos chain tailored towards performance and scaleability to use funds that originate on Ethereum and possibly record events in a Corda distributed ledger or, in the reverse, a Corda ledger initiating the transfer of underlying assets defined in Cosmos or Ethereum.
+
+New concepts such as Interchain Accounts will continue to further increase adoption times and the possibilities IBC offers: a blockchain will be able to control an account on another blockchain with Interchain Accounts.
```

Comment:
also, not sure if this makes sense without first addressing the two layers of IBC: transport and application. ICA is an example of a new application module, whereas other transport layer features to come will be stuff like partially ordered channels, multihop routing, etc.